### PR TITLE
Modify auth config route to include additional values

### PR DESCRIPTION
**Pull request by var-ram**
Changed key names and removed algorithm from config response
Added auth config spec

### CODE REVIEW

File: specs/auth/spec.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+openapi: 3.0.0
```
Comment (gcarwardine) [inline specs/auth/spec.yaml:1]: Not convinced this file should be in this project. The "servers" defined are specific to the EWB server. At the very most it should be a template other projects can pull in to add to their own swagger specs.

**Review by gcarwardine — CHANGES_REQUESTED** [review]
Other than my comment on the spec file this looks fine to me.
I'll let @kgreav comment on the alg key being removed on the response (not sure if he was relying on this at alll.)

**Comment by var-ram** [comment]
> Other than my comment on the spec file this looks fine to me.
> I'll let @kgreav comment on the alg key being removed on the response (not sure if he was relying on this at alll.)

Already spoke to Kurt about it. He was happy for me to remove it.

File: specs/auth/spec.yaml (hunk 1 above)
Comment (var-ram) [reply specs/auth/spec.yaml:1]: Yeaup I just put it there thinking the `evolve-conn-jvm` lib was similar to the `ewb-network-routes` but its more reusable I guess. Will change it to a template like you suggested and have the api spec under ewb server

**Review by gcarwardine — APPROVED** [review]

**Pull request merged**